docs/products/opensearch/concepts/index-usage.rst

```diff
@@ -0,0 +1,36 @@
+Index usage
```

Comment:
Can we make the title more clear about the content? Like 

"Indexes design" or "indexes usefulness" or something along the lines of understanding if an index is useful?

Reply:
I renamed it to "When to create a new index"